Sources/OperatingSystem.swift

```diff
@@ -26,4 +26,213 @@
     @_exported import Glibc
 #else
     @_exported import Darwin.C
-#endif
\ No newline at end of file
+#endif
+
+import C7
+
+public enum SystemError: ErrorProtocol {
+    
```

Comment:
> errno.h - system error numbers

(http://pubs.opengroup.org/onlinepubs/009695399/basedefs/errno.h.html)
So I think `SystemError` is fine :)